ui/app/src/utils/i18n-legacy.ts

```diff
@@ -690,17 +691,18 @@ export const browseSearchMessages = defineMessages({
 });
 
 export const formEngineMessages = defineMessages({
+  inProgressConfirmation: {
+    id: 'formEngine.inProgressConfirmation',
+    defaultMessage:
+      'Another editor is currently open & minimized. Please close the current editor before opening another'
```

Comment:
This is a form. The other is an editor. This was was fine saying form in both sentences. The one above should say editor in both sentences.